section11/player.py

```diff
@@ -0,0 +1,27 @@
+from hand import Hand
+from coins import Coins
+
+
+class Player:
+
+    def __init__(self):
+        self.hand = Hand()
+        self.coins = Coins()
+
+    def make_bet(self):
+        while True:
+            try:
+                bet = int(input("Make a bet: "))
+
+                # Check if the bet is within the valid range
+                if bet <= 0:
+                    print('Bet must be greater than 0.')
+                elif bet > self.coins.total:
+                    print(f'Not enough coins! You have {self.coins.total} coins.')
+                else:
+                    # Set the bet and break out of the loop
+                    self.coins.bet = bet
+                    print(f'Your bet is {bet}!\n')
+                    break
+            except ValueError:
```

Comment:
with 'try and except', we try not to specify the part we expect an error. I think you need to put it in line 15